python/pyiceberg/expressions/literals.py

```diff
@@ -381,10 +381,12 @@ def __init__(self, value: Decimal):
         super().__init__(value, Decimal)
 
     def increment(self) -> Literal[Decimal]:
-        return DecimalLiteral(self.value + 1)
+        inc_value = 1 / (10 ** abs(self.value.as_tuple().exponent))
```

Comment:
We don't want to do decimal math like this because we then have to worry about precision. Instead, can you use the to/from unscaled methods?

Reply:
It may also make sense to have this in `util/decimal.py` instead of here.

Reply:
Ah,  wasn't aware that we already had those methods around. I've used those instead 👍🏻